src/west/main.py

```diff
@@ -390,6 +390,8 @@ def set_zephyr_base(args):
         zb_prefer = config.config.get('zephyr', 'base-prefer',
                                       fallback=None)
         zb_config = config.config.get('zephyr', 'base', fallback=None)
+        if zb_config is not None and not os.path.isabs(zb_config):
+            zb_config = os.path.join(west_topdir(), zb_config)
```

Comment:
os.path.join(absolute_path1, absolute_path2) is just absolute_path2, so I think the check is unnecessary, no?

Reply:
but it only joining if `zb_config` is NOT abs path:
```
not os.path.isabs(zb_config)
```

Reply:
ahh, misread your comment
You mean `not os.path.isabs(zb_config)` can be removed

Will remove when I have verified, that it indeed works that way.

Reply:
Fixed